bindings/python/tv.cc

```diff
@@ -125,6 +125,7 @@ void tvFromJson(const std::vector<std::string>& input_json_per_rank_list, const
     render.generate(font_size, win_size);
   } catch (std::exception const& e) {
     std::cout << "vt-tv: Error reading the configuration file: " << e.what() << std::endl;
+    exit(1);
```

Comment:
Is `exit(1)` appropriate here? Isn't the point of exceptions not to exit but to allow them to be caught?

Reply:
Hi @lifflander I added this because the exception was not re-raised after the message.
I discovered that because in CI the python test was passing but there was an error in fact.

But there is also the possibility to remove the catch statement or to reraise the error. The goal is here to get a status code that is not zero to say to the caller that there is a problem.

Reply:
Let's throw an exception

Reply:
Fixed